docs/evolve_spec.rst

```diff
@@ -90,19 +90,13 @@ Backwards Compatible Changes
     data type and continue to present a tag with no value to the
     application.
 
-* Adding another tag to a union that has a catch-all specified.
+* Adding another tag to an open union.
 
   * The older receiver will not understand the incoming tag, and will
     simply set the union to its catch-all tag. The application-layer will
     handle this new tag through the same code path that handles the
     catch-all tag.
 
-Planning for Backwards Compatibility
```

Comment:
I know the default is an open `union` now, but I think it's still worth calling out the difference, and how the developer should choose between the two. So, let's keep this section but explain when one should use open or closed.